packages/neuron-wallet/src/utils/parse_sudt_token_info.ts

```diff
@@ -0,0 +1,18 @@
+const parseSUDTTokenInfo = (hexData : string ) => {
+    let data = hexData
```

Comment:
I'm not quite sure about the encoding of `hexData`, but according to the test cases, I think the decoding can be simplified as follows:

```js
const decode = (hexString) => {
  const decimal = +hexString.substr(0, 4)
  const [_, name, symbol] = Buffer.from(hexString.slice(4), 'hex').toString('utf-8').split(`\n`)
  return { decimal, name, symbol }
}
```

Reply:
good idea